README.md

```diff
@@ -36,14 +36,20 @@ This project was created with Docker version 20.10.5.
 
 1. Install [Docker](https://docs.docker.com/get-docker/)
 2. Clone this repository
-3. If running for the first time, and no local database exists, a new one wil be built for you based on `/db/db.sql`. If you already have a local database you will need to update the values for `DB_PASSWORD` and `SECURITY_KEY` in `docker-compose-local-db.yml` to use that database.
-4. Bring the Docker compose file up:
+3. Add a .env file (based on .env.sample) and provide values appropriate to your local dev environment
```

Comment:
Out of curiosity, what's the impetus for using the .env over just modifying the `docker-compose` yaml files?

Reply:
because docker-compose files have to be tracked in git but .envs should never be tracked.

Reply:
Fair enough